lib/queuePopulator/IngestionProducer.js

```diff
@@ -60,10 +57,9 @@ class IngestionProducer {
         this.sourceHTTPAgent = new http.Agent({ keepAlive: true });
 
         this.ringReader = new BackbeatClient({
-            endpoint: 'http://' +
-                `${this.s3source.host}:${this.s3source.port}`,
+            endpoint: `${protocol}://${host}:${port}`,
             credentials: this.s3sourceCredentials,
-            sslEnabled: this.qpConfig.transport === 'https',
+            sslEnabled: sourceConfig.auth.https,
```

Comment:
`protocol === 'https'`, wouldn't this be sufficient so that we can remove `sourceConfig.auth.https`?

Reply:
good call 👍 updated